src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  const basicPrice = days * oneDayRent;
+
+  if (days >= 3 && days < 7) {
+    return basicPrice - 20;
+  }
+
+  if (days >= 7) {
+    return basicPrice - 50;
+  }
```

Comment:
Swap these blocks and you won't have to duplicate conditions.
if it haven't returned anything for `days >= 7`, it automatically means that `days < 7` is true